frontend/packages/client/src/components/common/Toast.js

```diff
@@ -0,0 +1,43 @@
+import { useHistory } from 'react-router-dom';
+import { Svg } from '@cast/shared-components';
+
+const Toast = ({ onClose, text, footerText, footerLink }) => {
+  const history = useHistory();
+
+  return (
+    <div
+      className="has-bg-white rounded-sm-br rounded-sm-tr border-left-yellow box-shadow p-2"
+      style={{ position: 'relative', width: 325, top: -25 }}
+    >
+      <div
+        className="cursor-pointer"
+        onClick={onClose}
+        style={{
+          position: 'absolute',
+          top: 8,
+          right: 8,
+        }}
+      >
```

Comment:
trying to use less bulma so went with absolute position close vs the `columns` spaghetti we usually have to create